system-test/spanner.ts

```diff
@@ -37,6 +37,7 @@ import {google} from '../protos/protos';
 import CreateDatabaseMetadata = google.spanner.admin.database.v1.CreateDatabaseMetadata;
 import CreateBackupMetadata = google.spanner.admin.database.v1.CreateBackupMetadata;
 
+const SKIP_BACKUPS = process.env.SKIP_BACKUPS;
```

Comment:
Minor nit on the name.

```suggestion
const SKIP_BACKUP_TESTS = process.env.SKIP_BACKUP_TESTS;
```